servers/curated-corpus-api/src/database/mutations/ApprovedItem.ts

```diff
@@ -137,6 +141,9 @@ export async function deleteApprovedItem(
     },
   });
 
+  // delete all SectionItems based on this ApprovedItem
+  await deleteSectionItemsByApprovedItemId(db, approvedItem.id);
```

Comment:
I think it would be good to add test coverage to this line, to prevent a future dev from accidentally removing or breaking it.

Reply:
good idea! added a commit.